interface/patient_file/report/patient_report.php

```diff
@@ -1,10 +1,35 @@
 <?php
+/**
+ *
+ * Comprehensive Patient Report
+ *
+ * Copyright (C) 2016 OpenEMR
```

Comment:
Is there a default/standard copyright template - specially for the original code taken over?